apps/web/src/services/report.ts

```diff
@@ -90,7 +90,7 @@ const createSheet = (
   creator: ReportCreator,
   filter: PeriodFilter,
 ): WorkSheet => {
```

Comment:
regardless of the code smell, I feel like this function is getting quite complex

Reply:
☝️ Agreed